src/main.rs

```diff
@@ -62,7 +62,7 @@ fn main() {
 
     debug!("Scripts initialized {:?}", &scripts_to_run);
 
-    if !opts.greppable && !opts.accessible {
+    if !opts.greppable && !opts.accessible && !opts.no_banner {
```

Comment:
As this line rather shows, there are already flags which will suppress the banner: is there something specific this change makes that the existing flags don't cover?

Reply:
Using `--greppable` gives us an array like output and `--accessible` gives us an output without the banner but it lacks the colored output which is quite useful. `--no-banner` shows the default colored output without the banner.